core/src/test/scala/integration/kafka/api/AbstractConsumerTest.scala

```diff
@@ -412,13 +413,15 @@ abstract class AbstractConsumerTest extends BaseRequestTest {
    * 1. Every consumer got assigned at least one partition
    * 2. Each partition is assigned to only one consumer
    * 3. Every partition is assigned to one of the consumers
+   * 4. The assignment is the same as expected assignment (if provided)
```

Comment:
Before the change, we don't verify the assignment results are the expected distribution. It's fine since we already tested that in each assignor tests. But here, I added a default assignor test, that I wanted to make sure it is using our expected assignor (i.e. Range assignor) to distribute the partitions.